Unit-02/01-blueprints/project/messages/templates/messages/edit.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'base.html' %}
```

Comment:
small thing but the indentation is a little off in this file, some lines appear to be overindented.